File: Python Word Examples/BookmarksApiDemo/deleteBookmark.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.bookmarks_api.BookmarksApi(configuration)

file_name = 'Bookmarks.docx'
bookmark_name = 'Test'
extra = {'folder': 'input',
         'dest_file_path': 'output/DeleteBookmark.docx'}
api.delete_bookmark(file_name, bookmark_name, **extra)

File: PHP Word Examples/BookmarksApiDemo/deleteBookmark.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\BookmarksApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new BookmarksApi($configuration);

$name = "Bookmark.docx";
$bookmark_name = 'Test';
$folder = "input";
$storage = null;
$password = null;
$dest_file_path ="output/DeleteBookmarkForText.docx";
$result = $apiInstance->deleteBookmark(
    $name,
    $bookmark_name,
    $dest_file_path,
    $folder,
    $storage,
    $password
);